scripts/winesapos-snap.sh

```diff
@@ -0,0 +1,5 @@
+git clone https://aur.archlinux.org/snapd.git
+cd snapd
+makepkg -si
```

Comment:
We already build `snapd` here:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/repo/winesapos-build-repo.sh#L63

and install it here:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/winesapos-install.sh#L846